src/sentry/static/sentry/app/views/performance/transactionSummary/content.tsx

```diff
@@ -204,22 +202,27 @@ class SummaryContent extends React.Component<Props, State> {
               eventView={eventView}
               totalValues={totalCount}
             />
-            <TransactionsList
-              location={location}
-              organization={organization}
-              eventView={eventView}
-              selected={selectedSort}
-              options={sortOptions}
-              titles={[t('id'), t('user'), t('duration'), t('timestamp')]}
-              handleDropdownChange={this.handleTransactionsListSortChange}
-              generateLink={{
-                id: generateTransactionLink(transactionName),
-              }}
-              baseline={transactionName}
-              handleBaselineClick={this.handleViewDetailsClick}
-              handleCellAction={this.handleCellAction}
-              handleOpenInDiscoverClick={this.handleDiscoverViewClick}
-            />
+            {totalValues === null ? (
+              <Placeholder height="352px" />
```

Comment:
@alxu-sentry This placeholder is far too large. It's preferable to delegate the placeholder to the `<TransactionsList />` component. It already has a decent loading state from the discover query component inside. That would provide better loading aesthetics.

Reply:
addressed this issue in https://github.com/getsentry/sentry/pull/24298/commits/86f9e82ab5c00351813e64f719859de5cba84ae6